pykickstart/options.py

```diff
@@ -126,11 +126,14 @@ def __init__(self, *args, **kwargs):
         """
         # Overridden to allow for the version kwargs, to skip help option generation,
         # and to resolve conflicts instead of override earlier options.
-        version = kwargs.pop("version", None)
+        version = kwargs.pop("version") # fail fast if no version specified
```

Comment:
@bcl, @clumens - I'm using this fail fast approach at a few places. Basically fail if no version/help parameters are provided. This will cause failures b/c there are commands and arguments for which I don't have documentation to copy.